tests/e2e/e2e_workflow.spec.js

```diff
@@ -78,8 +87,7 @@ describe('Basic end-to-end Workflow', function () {
       .split('\n');
 
     // Check that a few critical files have been detected.
-    expect(lines).to.include(`${path.sep}index.html`);
-    expect(lines).to.include(`${path.sep}thirdparty${path.sep}libs.js`);
+    //expect(lines).to.include(`${path.sep}index.html`);
```

Comment:
Forgot about that to fix, `libs.js` are removed. Still not sure how can tests passed before.